ci/ci/prs.py

```diff
@@ -428,6 +428,7 @@ def merge(self, pr):
         if status_code == 200:
             log.info(f'successful merge of {pr.short_str()}')
             self._set(pr.source.ref, pr.target.ref, pr.merged())
+            pr.notify_github(pr.build, status_sha=gh_response['sha'])
```

Comment:
it would be nice if it also included a target_url that linked to `https://storage.googleapis.com/{GCS_BUCKET}/deploy/status_sha/index.html`.

Reply:
can we do that as an incremental add once this is working?